functions/contains.yml

```diff
@@ -1,54 +1,84 @@
 name: contains
-topic:
+topic: collections
 args:
   required:
-  - name: subject,
+  - name: subject
     type: [array, string]
-    desc:
+    desc: ""
   - name: search
     type: [any]
-    desc:
+    desc: ""
   optional: []
 returns:
   type: boolean
-  desc:
+  desc: ""
 desc: |
   Returns `true` if the given `$subject` contains the provided `$search`
   string.
-  
+
   If `$subject` is an array, this function returns true if one of the elements
   in the array is equal to the provided `$search` value.
-  
+
   If the provided `$subject` is a string, this function returns true if
   the string contains the provided `$search` argument.
 examples:
-- context: None
+- context:
   args: [foobar, foo]
   returns: true
-- context: None
+- context:
   args: [foobar, not]
   returns: false
-- context: None
+- context:
   args: [foobar, bar]
   returns: true
-- context: None
+- context:
   args: [false, bar]
-  returns: "<error: invalid-type>"
-- context: None
+  error: invalid-type
+- context:
   args: [foobar, 123]
   returns: false
-- context: ["a", "b"]
-  args: [@, a]
+- context: [a, b]
+  args: ['@', a]
   returns: true
-- context: ["a"]
-  args: [@, a]
+- context: [a]
+  args: ['@', a]
   returns: true
-- context: ["a"]
-  args: [@, b]
+- context: [a]
+  args: ['@', b]
   returns: false
-- context: ["foo", "bar"]
-  args: [@, foo]
+- context: [foo, bar]
+  args: ['@', foo]
   returns: true
-- context: ["foo", "bar"]
-  args: [@, b]
+- context: [foo, bar]
+  args: ['@', b]
+  returns: false
+- context: &data
```

Comment:
Is this correctly formatted YAML syntax ?
There seems to be property / value pairs below, but the `&data` part looks like a value to `context`.

Reply:
this is a feature of yaml, it is a YAML anchor and refers to the following node
The following aliases (*data) refer back to that node.
This deduplicates data within the document, or otherwise specifies a node with multiple ancestors in the yaml digraph.